src/sparql/ordo-replace-annotation-based-mappings.ru.jinja2

```diff
@@ -0,0 +1,28 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix sssom: <https://mapping-commons.github.io/sssom/purl/>
+
+{% for mapping_precision_string, relationship_curie in mapping_str__curie__map.items() %}
+DELETE {
+  ?xref_anno a owl:Axiom ;
+    ECO:0000218 ?mapping_precision_string .
+} INSERT {
+  ?cls a owl:Class;
+      {{ relationship_curie }} ?value .
+} WHERE {
+  VALUES ?mapping { oboInOwl:hasDbXref }
+
+  ?cls a owl:Class;
+    ?mapping ?value .
+
+  ?xref_anno a owl:Axiom ;
+    owl:annotatedSource ?cls ;
+    owl:annotatedProperty ?mapping ;
+    owl:annotatedTarget ?value ;
+    ECO:0000218 ?mapping_precision_string .
+
+  FILTER(STRSTARTS(STR(?mapping_precision_string), "{{ mapping_precision_string }}"))
+};
```

Comment:
My IDE shows a syntax error near the end of my instantiated `.ru` file.

When there are multiple WHERE clauses, you can delimit them with a `;`. But my IDE is telling me that the final clause should not have a `;` at the end.

I'm not sure if this really matters, because I ran the query and it appears to have worked.

If I want to fix this (removing the final `;`), there's two ways:
a. In my instantiating Python file, I can, after creating the Jinja template, remove the final `;`. This would be easy, but this would mean that the Jinja template alone doesn't contain all the logic.
b. More complex: There is a way to do indexing when Jinja iterates over an iterable. If it is the final index, it can make sure to leave the `;` out. Iv'e done stuff like this in Jinja before, but it is a little complicated, and makes for a pretty messy looking Jinja file.

Of course, we could just ignore this, since it appears that the query worked.

Reply:
If it works, leave it